src/main/java/org/cascadebot/cascadebot/utils/pagination/PageCache.java

```diff
@@ -41,6 +43,13 @@ public int getPages() {
             return pages.size();
         }
 
+        public boolean isMissingButtons() {
+            return missingButtons;
```

Comment:
What is this used for?

Reply:
If the pages is missing buttons due to it being on the first page, or on the last.